Makefile

```diff
@@ -1174,44 +1174,30 @@ update-tag:
 	(cd e && git tag $(GITTAG) && git push origin $(GITTAG))
 	git push $(TAG_REMOTE) $(GITTAG) && git push $(TAG_REMOTE) api/$(GITTAG)
 
-# HAS_CLOUD_SEMVER is non-empty if $(VERSION) contains a cloud-only pre-release tag,
```

Comment:
For some reason this is reverting changes I made recently for cloud-only releases.

Reply:
My bad, I was trying to do some git fu to squash this to a single commit and goofed something. Should be resolved now.